### PR TITLE
Anna Terelius project5 Cluedo

### PR DESCRIPTION
**Pull request by AnnaTerelius**
https://annatereliusproject5cluedo.netlify.com/

### CODE REVIEW

File: code/clue.js [hunk 1]
```diff
@@ -1,64 +1,210 @@
...
+    age:"38",
+    image: "assets/plum.png",
+    occupation: "professor"
+  }
+ const missScarlet = {
+    firstName: "Cassandra",
+    lastName: "Scarlet",
+    color: " crimson",
+    description: "will do anything to stay in the sportight",
+    age:"25",
```
Comment (ericaglimsholt) [inline code/clue.js:26]: So an age is a number, which means it is an integer in code. Now you have written it as a string (when you have `"` around) which you do if you are typing out letters in some way. So in this case you can re write this as `age: 25,` (and all of the others integers)

File: code/clue.js [hunk 2]
```diff
@@ -1,64 +1,210 @@
...
 
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer.
-  const theKiller = document.getElementById("killer")
-  const theKillerName = document.getElementById("killerName")
+theKiller.style.background = mystery.killer.color
+theKillerName.innerHTML = mystery.killer.firstName + " " + mystery.killer.lastName
+thekillerDescription.innerHTML = mystery.killer.description
+thekillerAge.innerHTML = mystery.killer.age
+thekillerOccupation.innerHTML = mystery.killer.occupation
+thekillerImage.src = mystery.killer.image
```
Comment (ericaglimsholt) [inline code/clue.js:173]: Would like some indention here!

File: code/clue.js [hunk 3]
```diff
@@ -1,64 +1,210 @@
...
+const pickWeapon = () => {
+ mystery.weapon = randomSelector(weapons);
+ console.log(pickWeapon)
+     
+const theWeapon = document.getElementById("weapon")
+const theWeaponName = document.getElementById("weaponName")
+  
+theWeapon.style.background = mystery.weapon.color
+theWeaponName.innerHTML = mystery.weapon.name + " " + mystery.weapon.weight 
+theWeapon.style.background = mystery.killer.color
```
Comment (ericaglimsholt) [inline code/clue.js:186]: Indent

File: code/clue.js [hunk 4]
```diff
@@ -1,64 +1,210 @@
...
+const pickRoom = () => {
+ mystery.room = randomSelector(rooms);
+ console.log(pickRoom)
+
+const theRoom = document.getElementById("room")
+const theRoomName = document.getElementById("roomName")
+
+theRoom.style.background = mystery.room.color
+theRoomName.innerHTML = mystery.room 
+theRoom.style.background = mystery.killer.color
```
Comment (ericaglimsholt) [inline code/clue.js:198]: Indent

File: code/clue.js [hunk 5]
```diff
@@ -1,64 +1,210 @@
...
+}
+ 
+// The function revealMystery will be invoked when you click the button. It will show the murderer, weapon and room.
 
-// CREATE A FUNCTION revealMystery that will be invoked when you click that button. It should show something like:
-// 'The murder was committed by Jacob Green, in the living room with a rope.'
+const revealMystery = () => {
+if (mystery.killer === ""||mystery.weapon === ""||mystery.room === ""){
+  document.getElementById("mystery").innerHTML = `The mystery is yet to be revealed`  
+}
```
Comment (ericaglimsholt) [inline code/clue.js:206]: Nice! Good work with error handling

File: code/style.css [hunk 6]
```diff
@@ -33,26 +41,65 @@ p {
...
+
+h2 {
+  text-align: center;
+}
+
+footer {
+text-align: center;
+margin-top: 20px;
+border-top: 1px solid black;
+border-bottom:1px solid black ;
```
Comment (ericaglimsholt) [inline code/style.css:75]: Indent

**Review by ericaglimsholt — APPROVED** [review]
Good job Anna! Keep this up! 
One thing to think about is to work with indention (with tab or spaces) to make the code more readable. I understand that it's enjoying getting those comments everywhere in this assignment but too be truthfully that's a part that's very important when you applying to a new job (in programming). The recruiter/programmer will always look if your code is beautiful and easy to read (also why it's important with comments) :)

**Comment by AnnaTerelius** [comment]
Hej Erica!
Tack för responsen, tvärt om, jag blir glad över dina kommentarer, jag vill ju lära mig 😊
Jag försöker skriva koden som jag tror den ska vara men lyckas helt klart inte fullt ut!
Jag vet inte hur indention ska vara, jag får läsa på code guidelines och hoppas jag hittar svaren där.
Hoppas koden är finare nästa gång du ska läsa.
Ha en fin dag!
/Anna

Från: Erica Glimsholt <notifications@github.com>
Skickat: den 31 oktober 2019 10:09
Till: Technigo/project-cluedo <project-cluedo@noreply.github.com>
Kopia: AnnaTerelius <tereliusanna@hotmail.com>; Author <author@noreply.github.com>
Ämne: Re: [Technigo/project-cluedo] Anna Terelius project5 Cluedo (#27)


@ericaglimsholt approved this pull request.

Good job Anna! Keep this up!
One thing to think about is to work with indention (with tab or spaces) to make the code more readable. I understand that it's enjoying getting those comments everywhere in this assignment but too be truthfully that's a part that's very important when you applying to a new job (in programming). The recruiter/programmer will always look if your code is beautiful and easy to read (also why it's important with comments) :)

________________________________

In code/clue.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-cluedo%2Fpull%2F27%23discussion_r341021545&data=02%7C01%7C%7C45d6fa2cc4bb451f92a808d75de20511%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637081097645209656&sdata=tHWOcPMTYVMKtDdXQ4d3X%2BuI2K9UQdPmTziOekiaJ2U%3D&reserved=0>:

> +  }

+ const professorPlum = {

+    firstName: "Victor",

+    lastName: "Plum",

+    color: "purple",

+    description: "Master detective",

+    age:"38",

+    image: "assets/plum.png",

+    occupation: "professor"

+  }

+ const missScarlet = {

+    firstName: "Cassandra",

+    lastName: "Scarlet",

+    color: " crimson",

+    description: "will do anything to stay in the sportight",

+    age:"25",

So an age is a number, which means it is an integer in code. Now you have written it as a string (when you have " around) which you do if you are typing out letters in some way. So in this case you can re write this as age: 25, (and all of the others integers)

________________________________

In code/clue.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-cluedo%2Fpull%2F27%23discussion_r341021842&data=02%7C01%7C%7C45d6fa2cc4bb451f92a808d75de20511%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637081097645209656&sdata=anJdux7HGDqa%2F%2Fne1j8JR%2BivG4nFrTYoI%2F5RX5yIIJA%3D&reserved=0>:

> + mystery.killer = randomSelector(suspects);

+ console.log(pickKiller)

+

+// This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer.

+const theKiller = document.getElementById("killer")

+const theKillerName = document.getElementById("killerName")

+const thekillerDescription = document.getElementById("killerDescription")

+const thekillerAge = document.getElementById("killerAge")

+const thekillerOccupation = document.getElementById("killerOccupation")

+const thekillerImage = document.getElementById("killerImage")



-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer.

-  const theKiller = document.getElementById("killer")

-  const theKillerName = document.getElementById("killerName")

+theKiller.style.background = mystery.killer.color

+theKillerName.innerHTML = mystery.killer.firstName + " " + mystery.killer.lastName

+thekillerDescription.innerHTML = mystery.killer.description

+thekillerAge.innerHTML = mystery.killer.age

+thekillerOccupation.innerHTML = mystery.killer.occupation

+thekillerImage.src = mystery.killer.image

Would like some indention here!

________________________________

In code/clue.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-cluedo%2Fpull%2F27%23discussion_r341021903&data=02%7C01%7C%7C45d6fa2cc4bb451f92a808d75de20511%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637081097645219660&sdata=P9s%2BzhNvo%2BpMtpli3yz%2FIFfHpiWvL0F50%2Fvw8EjY6Kw%3D&reserved=0>:

> +const theWeapon = document.getElementById("weapon")

+const theWeaponName = document.getElementById("weaponName")

+

+theWeapon.style.background = mystery.weapon.color

+theWeaponName.innerHTML = mystery.weapon.name + " " + mystery.weapon.weight

+theWeapon.style.background = mystery.killer.color

Indent

________________________________

In code/clue.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-cluedo%2Fpull%2F27%23discussion_r341022019&data=02%7C01%7C%7C45d6fa2cc4bb451f92a808d75de20511%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637081097645219660&sdata=EH0DfBl2MKfcpLTfE%2FwKpgiQMtuUdp1D03uLauYTtRw%3D&reserved=0>:

> +const theRoom = document.getElementById("room")

+const theRoomName = document.getElementById("roomName")

+

+theRoom.style.background = mystery.room.color

+theRoomName.innerHTML = mystery.room

+theRoom.style.background = mystery.killer.color

Indent

________________________________

In code/clue.js<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-cluedo%2Fpull%2F27%23discussion_r341022216&data=02%7C01%7C%7C45d6fa2cc4bb451f92a808d75de20511%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637081097645219660&sdata=30qAV5q3YKAVBilzmfpftk5htMvExPiw4PodQCLXtkQ%3D&reserved=0>:

> +if (mystery.killer === ""||mystery.weapon === ""||mystery.room === ""){

+  document.getElementById("mystery").innerHTML = `The mystery is yet to be revealed`

+}

Nice! Good work with error handling

________________________________

In code/style.css<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FTechnigo%2Fproject-cluedo%2Fpull%2F27%23discussion_r341022359&data=02%7C01%7C%7C45d6fa2cc4bb451f92a808d75de20511%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637081097645229670&sdata=i0IvOREcBHGOVDZajNqjlD7tz0D0G7P4oeMxPFqVp%2F4%3D&reserved=0>:

> +text-align: center;

+margin-top: 20px;

+border-top: 1px solid black;

+border-bottom:1px solid black ;

Indent

**Comment by ericaglimsholt** [comment]
Vad skönt att höra! Indentionen kan man välja själv om man vill använda tab (ett klick på tabb tangenten) eller space (använda space tangenten). Så det indentering går ut på att att se vad som tillhör till vad. Så det blir som en herarki. Exempelvis:
```
<div class="my-awesome-div">
<p>
lalal this is my awesome paragraph
that's can be very long or small
</p>
</div>
```
Är enklare att läsa med indentering som detta:
```
<div class="my-awesome-div">
   <p>
      lalal this is my awesome paragraph
      that's can be very long or small
   </p>
</div>
```

**Pull request closed without merging**